packages/vitaminjs/bin/vitamin.js

```diff
@@ -10,5 +10,6 @@ program
     .command('build', 'Build server and client bundles').alias('b')
     .command('clean', 'Delete server and client builds').alias('c')
     .command('start', 'Build and start application server').alias('s')
+    .command('jest', 'Launch test suite with Jest', { noHelp: true }).alias('j')
```

Comment:
what would happen if `vitamin jest` is invoke and vitamin-jest plugin is not yet installed?
Even if it throws, it will still appears in `vitamin --help`